civihr_employee_portal/civihr_leave_absences/civihr_leave_absences.module

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @file
+ * Module to add functionality for CiviHR Leave and Absences.
+ */
+
+/**
+ * Implements hook_block_info().
+ */
+function civihr_leave_absences_block_info() {
+  $blocks = [];
+  $blocks['my_leave'] = [
+    'info' => t('My Leave'),
+    'cache' => DRUPAL_NO_CACHE,
+  ];
+  return $blocks;
+}
+
+/**
+ * Implements hook_block_view().
+ *
+ * Generates block for Leave pages.
+ */
+function civihr_leave_absences_block_view($delta = '') {
+  $block = [];
+  switch ($delta) {
+    case 'my_leave':
+      $block['subject'] = t('My Leave');
+      $block['content'] = civihr_leave_absences_my_leave_get_markup();
+      break;
+  }
+  return $block;
+}
+
+/**
+ * Internal function for fetching custom markup from template/Angular Js to display My Leave block.
+ * @return Custom markup from template file.
+ */
+function civihr_leave_absences_my_leave_get_markup() {
+
+  // @ TODO -> Pointing to Angular JS App once its ready for My Leave block.
```

Comment:
You mean statically? In that case no, we have a drupal module to programmatically load resources from a civihr extension